app/email/email.go

```diff
@@ -591,13 +591,18 @@ func (es *Service) NewEmailTemplateData(locale string) templates.Data {
 		organization = localT("api.templates.email_organization") + *es.config().EmailSettings.FeedbackOrganization
 	}
 
+	supportEmail := *es.config().SupportSettings.SupportEmail
+	if supportEmail == "" && *es.license().Features.Cloud {
+		supportEmail = " feedback-cloud@mattermost.com"
+	}
```

Comment:
We only want to hardcode the support email address to certain cloud emails (the one in the ticket) not all server email notifciations and also regardless of what the support email is configured. Should we do this inside of the template of that email specifically directly instead?

Reply:
Undid the changes.